backend/prompt_studio/prompt_studio_index_manager/models.py

```diff
@@ -78,3 +88,28 @@ class Meta:
                 name="unique_document_manager_profile_manager",
             ),
         ]
+
+
+# Function will be executed every time an instance of IndexManager is deleted.
+@receiver(post_delete, sender=IndexManager)
+def perform_vector_db_cleanup(sender, instance, **kwargs):
+    """Signal to perform vector db cleanup."""
+    logger.info("Performing vector db cleanup")
```

Comment:
@athul-rs is there any unique id from instance or sender that we can add here to associate this with a parent operation or something? Else this log is not very useful and I think we could remove it or make it debug

Reply:
Done, add prompt tool id to log